v3.3/glfw/build_cgo_hack.go

```diff
@@ -0,0 +1,17 @@
+// +build required
+
+package glfw
+
+// this file exists purely to prevent the golang toolchain from stripping
+// away the c source directories and files.
```

Comment:
Trivial: please can you document the circumstances under which the files are stripped? :)

```suggestion
// This file exists purely to prevent the golang toolchain from stripping
// away the c source directories and files when `go mod vendor` is used
// to populate a `vendor/` directory of a project depending on `go-gl/glfw`.
```